### ISSUE TITLE
Writing the evaluation of UML diagrams

### ISSUE THREAD

**Issue by Goko6** [bounswe/bounswe2022group3#142]
### Task:
As stated in issue #132 :  This evaluation will be written on UML diagrams and reviewing their creation process
### Place To Put The Documentation:
Milestone 1 Report (After its done there will be a link here)
### Deadline:
14.04.2022 20.00

**Comment by Goko6** [comment]
The first draft of the evaluation is as follows:

We started creating our UML diagrams without deciding on a template because we thought that it would be easier if we reviewed all the work together at the meetings instead of spending so much time on creating a template for something that could differentiate way too often since different use cases or different sequence diagrams require a totally distinct approach. For use case and class diagrams we used lucidchart, however in sequence diagrams some of us found it easier to create sequence diagrams using plantUML whereas the rest remained to create theirs with lucidchart. We did not need to alter the use case diagrams too much but for class and sequence diagrams, we needed to modify them several times. We asked plenty of questions to the TA during our customer meetings and we finalized our design accordingly.
-	Use Case Diagrams: We split the use cases to every group member so that all of us would contribute to the creation of the diagram. We did not need to alter the use case bubbles all that much, but we did modify the arrows between them a lot since the initial creating had plenty of mistakes.
-	Class Diagrams: Like use case diagrams, we also split the class and sequence diagrams between us. Our initial creation looked somewhat successful but as we started creating our sequence diagrams, we found out that some of the needed functions were missing. We modified the class diagrams accordingly and we also did not include any getter-setter functions in our class diagrams since that we asked whether if we should add them or not and the answer we got was that we better not add them since they are not necessary to show.
-	Sequence Diagrams: We decided that everyone should create the sequence diagram of the use case they made. In the customer meeting, we received feedback to our drafts, and the feedback told us that our initial creations are a little complicated and that we need to simplify our sequence diagrams. We did the necessary simplifications and finalized our sequence diagrams.

**Comment by ersoykadir** [comment]
> -	Sequence Diagrams: ... In the customer meeting, we received feedback to our drafts, and the feedback told us that our initial creations are a little complicated and that we need to simplify our sequence diagrams ... 

This can be modified as;
> - Sequence Diagrams: ... In the customer meeting, we received feedbacks to our drafts and it was mainly about the fact that our initial creations are a little complicated and that we need to simplify our sequence diagrams ... 

Rest of the evaluation looks good.

**Comment by Goko6** [comment]
The second draft is ready with the feedback from @ersoykadir:

We started creating our UML diagrams without deciding on a template because we thought that it would be easier if we reviewed all the work together at the meetings instead of spending so much time on creating a template for something that could differentiate way too often since different use cases or different sequence diagrams require a totally distinct approach. For use case and class diagrams we used lucidchart, however in sequence diagrams some of us found it easier to create sequence diagrams using plantUML whereas the rest remained to create theirs with lucidchart. We did not need to alter the use case diagrams too much but for class and sequence diagrams, we needed to modify them several times. We asked plenty of questions to the TA during our customer meetings and we finalized our design accordingly.
-	Use Case Diagrams: We split the use cases to every group member so that all of us would contribute to the creation of the diagram. We did not need to alter the use case bubbles all that much, but we did modify the arrows between them a lot since the initial creating had plenty of mistakes.
-	Class Diagrams: Like use case diagrams, we also split the class and sequence diagrams between us. Our initial creation looked somewhat successful but as we started creating our sequence diagrams, we found out that some of the needed functions were missing. We modified the class diagrams accordingly and we also did not include any getter-setter functions in our class diagrams since that we asked whether if we should add them or not and the answer we got was that we better not add them since they are not necessary to show.
-	Sequence Diagrams: We decided that everyone should create the sequence diagram of the use case they made. In the customer meeting, we received feedbacks to our drafts and it was mainly about the fact that our initial creations are a little complicated and that we need to simplify our sequence diagrams. We did the necessary simplifications and finalized our sequence diagrams.
